pkg/goDB/conditions/node/filter_direction.go

```diff
@@ -0,0 +1,160 @@
+package node
+
+import (
+	"errors"
+	"fmt"
+	"github.com/els0r/goProbe/pkg/types"
+	"github.com/els0r/goProbe/pkg/types/hashmap"
+)
+
+// FilterTypeDirection denotes filters wrt. directionality of traffic
+type FilterTypeDirection string
+
+const (
+	filterTypeDirectionIn         FilterTypeDirection = "in"
+	filterTypeDirectionInSugared                      = "inbound"
```

Comment:
This is not an `iota`, only the first element in this `const` group will be of type `FilterTypeDirection`, the rest will be "untyped" strings.